src/Components/SmartComponents/Reports/ExecutiveReport.js

```diff
@@ -118,13 +118,13 @@ const ExecutiveReport = ({ data, label }) => {
                         <Column>
                         </Column>
                     </Section>
-                    <Section title="Top 3 CVEs in your infrastructure" withColumn={false}>
+                    <Section title={intl.formatMessage(messages.executiveReportTop3)} withColumn={false}>
                         {topCves.map(cve => (
                             <Panel key={cve.synopsis} title={cve.synopsis} description={cve.description}>
-                                <PanelItem title="CVSS score">
+                                <PanelItem title={intl.formatMessage(messages.cvssBaseScore)}>
```

Comment:
it's not the `messages.cvssBaseScore` in executive report we display `CVSS score` instead of `CVSS base score`